FABulous/fabric_files/FABulous_project_template_verilog/Test/Makefile

```
@@ -0,0 +1,57 @@
+MAX_BITBYTES=16384
+FAB_PROJ_ROOT=..
+DESIGN=sequential_16bit_en
+TESTBENCH=${DESIGN}_tb
+TOP_WRAPPER=top_wrapper
+
+BUILD_DIR=build
+FABRIC_FILES_DIR=${BUILD_DIR}/fabric_files
+USER_DESIGN_DIR=${FAB_PROJ_ROOT}/user_design
+USER_DESIGN_VERILOG=${USER_DESIGN_DIR}/${DESIGN}.v
+TOP_WRAPPER_VERILOG=${USER_DESIGN_DIR}/${TOP_WRAPPER}.v
+
+.PHONY: all run_FABulous_demo full build_test_design run_simulation_emulation mkdir_build clean
+
+sim: build_test_design run_simulation clean
+
+FAB_sim: run_FABulous_demo copy_files run_simulation clean
+
+full_sim: run_FABulous_demo build_test_design run_simulation_emulation clean
+
+build_test_design: run_yosys run_nextpnr run_bitgen copy_files
+
+run_simulation_emulation: run_emulation run_simulation
+
+run_FABulous_demo: mkdir_build
+	# Run FABulous, generate default fabric and generates bitstream for default user_design
```

```suggestion
	# Run FABulous, generate the default fabric and a bitstream for the default user_design
```

I would suggest this